src/store/utils.ts

```diff
@@ -123,14 +125,14 @@ export function combineData<
 }
 
 // Handle missing / data / error cases of a result and return a single value.
-export function handle<T, E, R>(
+export function handle<T, E>(
```

Comment:
Why these changes?

Reply:
Just a minor clean up while I'm in this file, there's not much value in a return type which doesn't depend on any input values.

Reply:
But these utils are generic and it is still nice to do `const data = handle(....)` and do something with data later, besides `ReactElement`.

Reply:
Reverted

Reply:
Not really reverted? Did you forget to push?

Reply:
Should be reverted now